package.json

```diff
@@ -303,7 +303,7 @@
         {
           "command": "mssql.createAzureFunction",
           "when": "view == objectExplorer && viewItem == Table",
-          "group": "MS_SQL@6"
+          "group": "Azure_Function@1"
```

Comment:
This should go below the other group - you can just name it `zAzure_Function@1` to do that

Reply:
updated!